adaptive/runner.py

```diff
@@ -287,9 +287,29 @@ def _submit(self, x):
         pass
 
 
-class BlockingRunner(BaseRunner):
+def BlockingRunner(
+    learner,
+    goal,
+    *,
+    executor=None,
+    ntasks=None,
+    log=False,
+    shutdown_executor=False,
+    retries=0,
+    raise_if_retries_exceeded=True,
+):
     """Run a learner synchronously in an executor.
 
+    **This runner is deprecated**, replace:
+
+    >>> r = adaptive.BlockingRunner(...)
+
+    with
+
+    >>> import asyncio
+    >>> r = adaptive.Runner(...)
+    >>> asyncio.run(r)
```

Comment:
This is wrong for a couple of reasons:

1. `Runner` is not a coroutine, so we can't use it with `asyncio.run`
2. `r.task` is a *task* and hence tied to a given ioloop.

This really should look like

```suggestion
    >>> asyncio.get_event_loop().run_until_complete(r.task)
```